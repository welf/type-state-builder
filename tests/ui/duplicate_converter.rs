use type_state_builder::TypeStateBuilder;

#[derive(TypeStateBuilder)]
struct TestStruct {
    #[builder(converter = |value: String| value.to_uppercase(), converter = |value: String| value.to_lowercase())]
    name: String,
}

