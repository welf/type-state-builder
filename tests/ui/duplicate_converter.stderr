error: Duplicate converter attribute. Only one converter is allowed per field
 --> tests/ui/duplicate_converter.rs:5:65
  |
5 |     #[builder(converter = |value: String| value.to_uppercase(), converter = |value: String| value.to_lowercase())]
  |                                                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/ui/duplicate_converter.rs:7:2
  |
7 | }
  |  ^ consider adding a `main` function to `$DIR/tests/ui/duplicate_converter.rs`
